src/drt/src/TritonRoute.cpp

```diff
@@ -1266,10 +1266,9 @@ void TritonRoute::reportDRC(const string& file_name,
               drcRpt << "bterm:" << bterm->getName() << " ";
               break;
             }
-            case frcInstBlockage: {
-              frInstBlockage* instBlockage
-                  = (static_cast<frInstBlockage*>(src));
-              drcRpt << "inst:" << instBlockage->getInst()->getName() << " ";
```

Comment:
Why remove this case?

Reply:
because it would never happen. Previously the gcNet of a macro obstruction was owned by the frInstBlockage. That made obstructions of the same macro be treated as different objects which made max-rectangles extraction wrong.
Now I make frInst the owner of all macro obstructions of that frInst. This corrects the max-rectangles extraction process which helps in detecting the undetected violations reported in https://github.com/The-OpenROAD-Project/OpenROAD/issues/2892